src/e2e-test/features/BQExecuter/BQExecute.feature

```diff
@@ -0,0 +1,64 @@
+@BQExecute
+Feature: BigQueryExecute - Verify successful data transfer to BQ table with BQExecute plugin
+
+  @BQ_SOURCE_TEST @BQ_SINK_TEST
+  Scenario: Verify data transfer of BigQueryExecute plugin with mandatory fields
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Enter the BigQueryExecute property sql "bqExecuteQuery"
+    Then Enter BigQueryExecute property datasetProjectId "projectId"
+    Then Enter BigQueryExecute property dataset "dataset"
+    Then Enter BigQueryExecute source property table name
+    Then Validate "BigQuery Execute" plugin properties
+    Then Close the BigQueryExecute Properties
+    Then Save the pipeline
+    Then Deploy the pipeline
```

Comment:
Add preview step
and steps to capture preview logs

same comment applies to all scenarios

Reply:
@sawantpritam -done

Reply:
Preview run and steps to capture preview logs still not added

Reply:
Then Save the pipeline
    Then Preview and run the pipeline
    Then Wait till pipeline preview is in running state
    Then Open and capture pipeline preview logs
    Then Verify the preview run status of pipeline in the logs is "succeeded"
    Then Close the pipeline logs
    Then Close the preview
    Then Deploy the pipeline
    Then Run the Pipeline in Runtime
    Then Wait till pipeline is in running state
    Then Open and capture logs
    Then Verify the pipeline status is "Succeeded"